### PR TITLE
Feature/ADF-207/Assign permissions to new item management roles

### PR DESCRIPTION
**Pull request by shpran**
**Relates to:** _[ADF-207](https://oat-sa.atlassian.net/browse/ADF-207)_

**Changes:** configured `Item Previewer` role.

**Require PR:** _[oat-sa/extension-tao-item#476](https://github.com/oat-sa/extension-tao-item/pull/476)_

### CODE REVIEW

File: migrations/Version202106221148456771_taoQtiTestPreviewer.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    ];
+
+    public function getDescription(): string
+    {
+        return 'Assign permissions to Item Previewer role';
+    }
+
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
```
Comment (gabrielfs7) [inline migrations/Version202106221148456771_taoQtiTestPreviewer.php:50]: Why do we need it here? The ontology should be updated in the extension it was changed, no?

https://github.com/oat-sa/extension-tao-item/pull/476/files#diff-f02c6f82cbea19dae3f9361e3e672b119d353a3de4082dd6d64183825c9f2fc8R153

```suggestion
```

**Review by gabrielfs7 — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: migrations/Version202106221148456771_taoQtiTestPreviewer.php (hunk 1 above)
Comment (gabrielfs7) [reply migrations/Version202106221148456771_taoQtiTestPreviewer.php]: Removed, thanks!

File: composer.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
     "oat-sa/generis" : ">=14.0.0",
     "oat-sa/tao-core" : ">=47.0.0",
-    "oat-sa/extension-tao-item" : ">=11.0.0",
+    "oat-sa/extension-tao-item" : ">=11.5.0",
```
Comment (gabrielfs7) [inline composer.json:33]: - [ ] Double check before merge

File: manifest.php [hunk 3]
```diff
@@ -32,8 +32,31 @@
...
+        [
+            AccessRule::GRANT,
+            'http://www.tao.lu/Ontologies/TAOTest.rdf#TaoQtiTestPreviewerRole',
+            ['ext' => 'taoQtiTestPreviewer'],
+        ],
+        [
+            AccessRule::GRANT,
+            TaoItemsRoles::ITEM_AUTHOR,
+            ['ext' => 'taoQtiTestPreviewer', 'mod' => 'Previewer'],
+        ],
```
Comment (gabrielfs7) [inline manifest.php:44]: Just to understand. Why those 2 were not added to the migrations?

Comment (shpran) [reply manifest.php:44]: These roles were here before - I refactored a bit CS and replaced the deprecated constant with a suitable one 🙂

**Review by gabrielfs7 — APPROVED** [review]

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-shpran.playground.kitchen.it.taocloud.org:41481/

**Pull request merged**